packages/components/src/components/infinite-data-list/infinite-data-list.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import InfiniteLoader from 'react-virtualized/dist/es/InfiniteLoader';
+import DataList from '../data-list/data-list.jsx';
+
+const InfiniteDataList = ({
+    className,
+    data_list_className,
+    has_more_items_to_load,
+    items,
+    keyMapperFn,
+    loadMoreRowsFn,
+    onScroll,
```

Comment:
Can we add all props as PropTypes? In this and other files?

Reply:
Sure, I'm sure some will be missed but we'll catch those when I'll do the cleanup. 👍